docs/src/tutorial-loop-unwinding.md

```diff
@@ -79,5 +79,39 @@ Failed Checks: dereference failure: pointer outside object bounds
 
 Kani is now sure we've unwound the loop enough to verify our proof harness, and now we're seeing just the bound checking failures from the off-by-one error.
 
+## Ways to specify unwinding value
+
+Kani allows three ways of specifying the unwind value for a purticular harness. They can be set through the below ways-
+
+1. Unwind annotation. This sets the unwind value for the harness above which the annotation is added. Example -
+``` rust
+#[kani::proof]
+#[kani::unwind(3)]
+fn proof_harness() {
+    ...
+}
+```
+2. `--unwind` flag. This sets the unwind value for the entire file/crate on which kani or cargo-kani is called. Example -
```

Comment:
I would say it's a global unwinding value. Take a look at previous documentation to ensure we are consistent.

Reply:
I would further say "sets the default unwind value for..."